Development/StatsAnalysis/SPM_VBA_Stats/xASL_spm_GLM.m

```diff
@@ -51,14 +51,14 @@
 for iSet=1:x.S.nSets
     clear ExpectedNIIcount FoundNIIcount
     ExpectedNIIcount    = size(x.S.DATASETS{iSet},1);
-    FoundNIIcount       = length(xASL_adm_GetFileList(x.S.SPMdir, ['^Set' num2str(iSet) 'subject\d*\.(nii|nii\.gz)$'],'FPList',[0 Inf]));
+    FoundNIIcount       = length(xASL_adm_GetFileList(x.S.SPMdir, ['^Set' num2str(iSet) 'subject\d*\.nii$'],'FPList',[0 Inf]));

```

Comment:
Is this nii.gz added automatically inside getFileList? Even if there's $ at the end?

Reply:
Because if nii and nii.gz are detected, then this is not mentioned in the adm_GetFileList function header.

Reply:
Yes, it automatically does this, on line 75, it replaces \\.nii (with or without $) by (\\.nii|\\.nii\\.gz)